core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -126,6 +132,93 @@ class KafkaApisTest {
     )
   }
 
+  @Test
+  def testAuthorize(): Unit = {
+    val authorizer: Authorizer = EasyMock.niceMock(classOf[Authorizer])
```

Comment:
Do we need to specify the val type here? Since we are passing it to `niceMock`, doesn't it figure it out by itself? Same question for the next test.

Reply:
It does not compile without the type: `value authorize is not a member of Nothing` at L153.